cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -64,10 +64,25 @@ message Options {
 	// when a container requests a gpu
 	string GPUVHDPath = 10;
 
-        // scale_cpu_limits_to_sandbox indicates that container CPU limits should
-        // be adjusted to account for the difference in number of cores between the
-        // host and UVM.
-        bool scale_cpu_limits_to_sandbox = 11;
+	// scale_cpu_limits_to_sandbox indicates that container CPU limits should
+	// be adjusted to account for the difference in number of cores between the
+	// host and UVM.
+	bool scale_cpu_limits_to_sandbox = 11;
+
+	// default_container_scratch_size_in_gb is the default scratch size (sandbox.vhd) 
```

Comment:
nit: Let us make sure we use `sandbox.vhdx` everywhere.

Reply:
Good catch, updating